README.md

```diff
@@ -8,7 +8,8 @@ That is, it will read in following order.
 
 ## Supported platforms
 
-It should work on every OS if we build Secp256k1 for every platform (right now we only provide Linux binaries).
+* By default, the solution is built with dependencies on Secp256k1.Net and NSec (which have native dependencies that need to be built for every platform, and we right now only provide Linux binaries). This version is published as `DotNetLightning.Core` on Nuget.
+* For a more portable version (but possibly less performant) you can compile with BouncyCastle support, passing /p:BouncyCastle=True to your build. This version is published as `DotNetLightning` on Nuget.
```

Comment:
I think the package name is confusing. My suggestion is

* for libsodium/nsec version ... `DotNetLightning.Core` or `DotNetLightning.Core.NSec` (or both by alias if possible )
* for BouncyCastle version ... `DotNetLightning.Core.BouncyCastle`